project/pom.xml

```diff
@@ -592,6 +594,30 @@
                 <package>com.jcraft.jsch.agentproxy</package>
               </packages>
             </exception>
+            <!--
+              Due to the following issues:
+                https://issues.apache.org/jira/browse/MSHADE-206
+                https://issues.apache.org/jira/browse/MNG-5899
+              The contents of the shaded gson jar are directly included as part of jclouds-core.
```

Comment:
I don't think it makes sense to force people to use an old version of Maven just because of an issue in the Maven Shade plugin. It is a warning of the duplicate class finder, but in this case it's safe because the offending classes are exactly the same (as reported by the plugin too).

Reply:
> I don't think it makes sense to force people to use an old version of Maven just because of an issue in the Maven Shade plugin

Fair point. In that case, would a TODO that this can be removed once the issues are fixed make sense? Just so it's clear in future that this shouldn't be needed and can be removed safely?

Reply:
Yep, I added the TODO in the last commit before merging

Reply:
Thanks!